### PR TITLE
Removed git conflict code

### PR DESCRIPTION
**Pull request by jimmynotjim**
Removed some missed git conflict code in the isocon template.

### CODE REVIEW

File: identity/isocons.md [hunk 1]
```diff
@@ -221,11 +221,7 @@ This library comprises all currently approved isocons and will continue to expan
 
 #### Owning a home
 
-<<<<<<< HEAD
 ![Owning a home isocon]({{ site.baseurl }}/static/img/isocons/Isocons26.png "Owning a home")
-=======
-![Owning a home isocon]({{ site.baseurl }}/assets/img/isocons/Isocons26.png "Owning a home isocon")
->>>>>>> 4a8004254bdb072fa934a85d05e5b34fed64462c
```
Comment (Scotchester) [inline identity/isocons.md:228]: I think the version with " isocon" is what should be kept, to be consistent with all the rest.

Comment (jimmynotjim) [inline identity/isocons.md:228]: @Scotchester You sure? That's not the correct url

Comment (Scotchester) [inline identity/isocons.md:228]: Huh? The two URLs are identical.

Comment (jimmynotjim) [inline identity/isocons.md:228]: Look again. `/static/` vs `/assets/` also, there's no "isoncon" in these titles https://cfpb.github.io/design-manual/identity/isocons.html

Comment (Scotchester) [inline identity/isocons.md:228]: Oh, hah wow. Okay. Well, we should definitely use the correct URL. The quoted text (which creates the `title` and `alt` attributes), however includes "isocon" in ever case on the rest of the page but one.

Comment (jimmynotjim) [inline identity/isocons.md:228]: Ah that's just the attrs. Force updated so the commit log stays clean.

**Comment by Scotchester** [comment]
:+1: Thanks!

**Pull request merged**